config/scheduler-chromeos.yaml

```diff
@@ -80,7 +90,17 @@ _anchors:
       channel: node
       name: kbuild-gcc-10-x86-chromeos-stoneyridge
       result: pass
-    platforms: *amd-stoneyridge-platforms
+    platforms: *amd-platforms
+
+  test-job-x86: &test-job-x86
+    <<: *lava-job-collabora
+    event:
+      channel: node
+      name: kbuild-gcc-10-x86
+      result: pass
+    platforms:
+      - *amd-platforms
+      - *intel-platforms
```

Comment:
After digging a bit and trying various stuff, it seems there's no easy way to merge yaml arrays :/

I'd suggest to either:
* keep sepearate jobs for intel/amd/qcom/mtk devices
* create new anchors explicitly listing all arm64/x86 devices (yes, that's a lot of duplication, but it seems there's no other way)

Reply:
Thanks @a-wai for the heads up! I reworked the PR to define separate intel/amd/qcom/mtk jobs. To hopefully make the anchor definitions clearer, I also renamed the anchors for test jobs that use chromeos-specific configs to include a `chromeos` infix. Let me know if you think this works or if there are better ways to organize these anchors.

Reply:
Looks OK to me, maybe `s/stoneyridge/amd/` and `s/pineview/intel/` in the rest of the definitions for consistency?

Reply:
Ack - I'll add a commit to do that. 

I'm wondering if it would make sense to also `s/baseline-arm64-chromeos-mediatek/baseline-arm64-mediatek/` and `s/baseline-arm64-chromeos-qualcomm/baseline-arm64-qualcomm/`? given that the only difference between these test jobs is the kernel configuration used, I think they could probably have the same name (the user can always filter by defconfig if needed)

Reply:
I did once with multiple scheduler entries for the same job - it looks like working, so it might work.

Reply:
About the `chomeos` naming - I'm fine with keeping it too if it makes things easier to filter/track, my concern was just that we'd end up with a lot of duplication (e.g. every single tast test in this file will have 2 jobs, one based on the default config and the other one on the chromeos config). Fwiw having multiple scheduler entries for the same job seems to be working fine on staging - e.g.: `baseline-x86-amd` w/ `x86_64_defconfig+x86-board` fragment: https://staging.kernelci.org:9000/viewer?node_id=661578bdfbd84ee9bdef2688 and `baseline-x86-amd` w/  the chromeos fragment: https://staging.kernelci.org:9000/viewer?node_id=661491f4208996f1d40b2364 . Let me know which solution looks better to you, I'll try to apply it for both the x86 and arm64 use cases for consistency. 

Side note: `baseline-arm64-mediatek/qualcomm` introduced with this PR don't work due to a different issue, see https://github.com/kernelci/kernelci-pipeline/pull/486#discussion_r1553113532

Reply:
> my concern was just that we'd end up with a lot of duplication (e.g. every single tast test in this file will have 2 jobs, one based on the default config and the other one on the chromeos config)

Actually Tast tests are only meant to run on kernels built with the chromeos configs, so this is not an issue. Meaning those jobs don't actually require `chromeos` in their names, this only applies to kbuilds I believe, and of course test job templates/anchors.

Reply:
Right - so the baseline tests are likely the only ones that will be duplicated. I dropped the `chromeos` infix from the mediatek/qualcomm baseline jobs for consistency with the x86 baseline jobs, but I kept it on kbuild jobs and anchors - let me know if you prefer to keep it in the baseline job names as well. Otherwise, I think this PR should be ready to go.

Reply:
This should be fine that way, thanks!